app/controllers/pets_controller.rb

```diff
@@ -0,0 +1,52 @@
+class PetsController < ApplicationController
+  def index
+    @pets = Pet.all
+  end
+
+  def show
+    @pet = Pet.find(params[:id])
+  end
+
+  def new
+    @shelter = Shelter.find(params[:id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:id])
+    pet = Pet.new({
+      name: params[:name],
+      age: params[:age],
+      sex: params[:sex],
+      description: params[:description],
+      shelter_id: shelter.id,
+      image: params[:image],
+      status: 'Adoptable'
```

Comment:
also will add status to pets model so that it automatically is creating an 'adoptable' as default value for status